cmd/sealer/cmd/image/rmi.go

```diff
@@ -22,15 +22,27 @@ import (
 
 var removeOpts *options.RemoveImageOptions
 
+var exampleForRmiCmd = `sealer rmi {imageName or imageID} ...
```

Comment:
An example should be a real and detailed example, rather than the command illustration. What you add seems not an example, but a command usage help content.

Reply:
just follow the rule by the code:
```golang
var exampleForInspectCmd = `sealer inspect {imageName or imageID}
sealer inspect --format '{{.OCIv1.Config.Env}}' {imageName or imageID}
`
```
may i change the example to a specific name or id?

Reply:
> just follow the rule by the code:
> 
> ```go
> var exampleForInspectCmd = `sealer inspect {imageName or imageID}
> sealer inspect --format '{{.OCIv1.Config.Env}}' {imageName or imageID}
> `
> ```
> 
> may i change the `rmi` example to a specific name or id?

Reply:
OK